### ISSUE TITLE
Duplicate ⌘␣ showing in KeyCastr

### ISSUE THREAD

**Issue by leshill** [keycastr/keycastr#163]
OSX 10.15.4
KeyCastr 0.9.8
Installed via brew

Pressing <command>-<space> to bring up spotlight shows double command spaces.

![KeyCastr](https://user-images.githubusercontent.com/3938/79150325-69f0e380-7d7d-11ea-9564-bde82aba20c4.gif)

**Comment by djfinch** [comment]
I have this as well (default visualiser). Using ␣+⌘ for zooming inside AcrobatDC.
![Untitled](https://user-images.githubusercontent.com/13107713/83998044-10153000-a960-11ea-95b9-39a9a15fd826.gif)

**Comment by akitchen** [comment]
Yep, this is annoying -- KeyCastr actually receives two separate events when ⌘␣ is pressed. Unclear as to why, but we'd need to special-case this in order to remove the unexpected behavior.

**Comment by akitchen** [comment]
I have noticed that issue does not happen on certain machines (arm64 or the new keyboard layout). It still occurs in older models. It seems to be hardware-specific and is probably not worth addressing as a special case.
